dm/config/checking_item.go

```diff
@@ -56,6 +60,10 @@ var AllCheckingItems = map[string]string{
 	BinlogDBChecking:             "binlog db checking item",
 	ConnNumberChecking:           "connection number checking item",
 	TargetDBPrivilegeChecking:    "privileges of target DB checking item",
+	// lightning prechecks
+	LightningEmptyRegionChecking:        "lightning empty region checking item",
+	LightningRegionDistributionChecking: "lightning region distribution checking item",
+	LightningDownstreamVersionChecking:  "lightning downstream version checking item",
```

Comment:
I remember we need to check the empty space for local backend. Is it necessary here?